.github/workflows/release-notification.yml

```diff
@@ -0,0 +1,19 @@
+name: Release Notification
+
+on:
+  release:
+    types: [published]
```

Comment:
Why is this `published` rather than `released`?

Reply:
Using [published] ensures that the workflow is triggered only when a release is published (i.e., created and released).

Reply:
The typical sequence is that a release is first "published" to make it available, and then it may be subsequently "released" after any necessary validation or approval steps have been completed.

Your concern is valid. Let me update all the PRs.